### PR TITLE
Remove calls to rummager

### PR DESCRIPTION
**Pull request by Rosa-Fox**
[Trello](https://trello.com/c/E7WbMTBT/485-stop-collections-publishing-to-rummager)

- No longer need to make calls to Rummager.
- The way in which `collections-publisher` formats are searched indexed has been changed so that search information is sent from `collections-publisher` -> `publishing-api` -> `rummager` as opposed to sending data to `rummager` directly from `collections-publisher`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,24 +18,21 @@ This is a Ruby on Rails application for internal use by GDS and some departmenta
...
 
-- Creating & editing topics and mainstream browse pages
+- Creating & editing topics and mainstream browse pages.
 - Curate the pages tagged to a topic or browse page into "curated lists". This means they'll be displayed in a group with a title. By default, the links are not curated and will appear with a 'A-Z' heading.
 
 ![Screenshot of curated and non-curated pages](docs/screenshot-curated-topics.png)
 
 Left, a [curated mainstream browse page](https://www.gov.uk/browse/childcare-parenting/childcare), right, an [uncurated browse page](https://www.gov.uk/browse/justice/prisons-probation).
 
-Collections publisher retrieves information about which content is tagged to a topic or browse page from [rummager](https://github.com/alphagov/rummager). It publishes the curated lists to the [publishing-api](https://github.com/alphagov/publishing-api).
```
Comment (brenetic) [inline README.md]: Could leave in the curated lists bit?

File: README.md [hunk 2]
```diff
@@ -18,24 +18,21 @@ This is a Ruby on Rails application for internal use by GDS and some departmenta
...
-This application uses the [sidekiq](http://sidekiq.org/) message queue for background work (mainly publishing to the publishing-api).
+This application uses the [sidekiq](http://sidekiq.org/) message queue for background work (mainly publishing to the Publishing API).
 
 ### Dependencies
 
 - [alphagov/publishing-api](https://github.com/alphagov/publishing-api) -
-  used to write curated lists.
-- [alphagov/rummager](https://github.com/alphagov/rummager) -
-  to index topics and mainstream browse pages, and to fetch documents that have
-  been tagged to topics and mainstream browse pages by publisher tools.
```
Comment (suzannehamilton) [inline README.md]: Does collections-publisher still need rummager to "fetch documents that have been tagged to topics and mainstream browse pages"? Or was this comment out of date?

Comment (Rosa-Fox) [reply README.md]: Hi, 

It was out of date. It was changed in this [commit](https://github.com/alphagov/collections-publisher/commit/93727c99e9833f726268ae49dbb7bf7da0411ca9).

File: README.md (hunk 1 above)
Comment (Rosa-Fox) [reply README.md]: @brenetic I took it out and explained below when describing the `publishing-api` dependency. Less repetition! Also the line below about sidekiq also makes it clear this app publishes to the publishing-api.

**Comment by Rosa-Fox** [comment]
@brenetic thanks. I left `rummager_format` thinking it needed to be sent to `publishing-api` but it isn't presented... because we map this in Rummager. Will remove. Thanks for spotting!

**Pull request merged**